examples/mnist/run.py

```diff
@@ -170,22 +166,27 @@ def receive_output(self, logits: tf.Tensor) -> tf.Operation:
       local_data_file="./data/test.tfrecord")
 
   # get model parameters as private tensors from model owner
-  params = tfe.define_private_input(model_owner.player_name,
-                                    model_owner.provide_input, masked=True)  # pylint: disable=E0632
+  params = tfe.define_private_variable(model_owner.player_name,
```

Comment:
I think it's okay to use `define_private_input` here

Reply:
sorry, I know I kept mentioning private_variable but the way @yanndupis 's code worked out it makes sense both ways and this one is more convenient because it can do masking automatically 😄

Reply:
I switched it to `define_private_variable` in the most recent commit as I encountered this error with `define_private_input`:

```W
  File "run.py", line 169, in <module>
    params = tfe.define_private_input(model_owner.player_name,model_owner.provide_input)
  File "/Users/vinithms/Desktop/Dropout Labs/tf-encrypted/tf_encrypted/protocol/pond/pond.py", line 570, in define_private_input
    factory=factory,
  File "/Users/vinithms/Desktop/Dropout Labs/tf-encrypted/tf_encrypted/protocol/pond/pond.py", line 534, in define_local_computation
    return [share_output(output) for output in outputs]
  File "/Users/vinithms/Desktop/Dropout Labs/tf-encrypted/tf_encrypted/protocol/pond/pond.py", line 534, in <listcomp>
    return [share_output(output) for output in outputs]
  File "/Users/vinithms/Desktop/Dropout Labs/tf-encrypted/tf_encrypted/protocol/pond/pond.py", line 476, in share_output
    enc = self._encode(v, apply_scaling, tf_int_type=factory.native_type)
  File "/Users/vinithms/Desktop/Dropout Labs/tf-encrypted/tf_encrypted/protocol/pond/pond.py", line 640, in _encode
    assert type(rationals) == type(integers)  # pylint: disable=unidiomatic-typecheck
AssertionError```

Reply:
I think that bug is a result of this problem: https://github.com/tf-encrypted/tf-encrypted/pull/578#discussion_r297451311